src/javascript/app/pages/trade/digit_trade.js

```diff
@@ -29,7 +30,20 @@ const DigitDisplay = (() => {
         }
     };
 
+    function setLoadingSpinner(content_id) {
+        $digit_table = $(`#${content_id}`);
+        $digit_table
+            .append($('<div />', { class: 'barspinner dark' })
```

Comment:
this isn't really what I meant. I meant moving this function to utility file etc so all pages can call it if needed. if we're still only specifically using it for digit table there's not much point in adding a separate function for it.

Reply:
I see what you mean now. This is a great idea. I generalize it a bit and move it to a separate module. (with show/hide functionality).